### PR TITLE
fix typo of shell

### PR DESCRIPTION
**Pull request by simonsmh**
fix typo of run.sh
https://github.com/PokemonGoF/PokemonGo-Bot/commit/e03f8341bb536dea3bc9753d705ea21e20a31418

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -15,7 +15,8 @@ while true
 do
 cd $pokebotpath
 python pokecli.py -cf $filename
-read -p "Press any button or wait 20 seconds." -r -s -n1 -t 20
-echo `date`"Pokebot"$*" Stopped." 
+read -p "Press any button or wait 20 seconds to continue.
```
Comment (TheSavior) [inline run.sh]: What is the typo here? Moving those flags to the next line seems like a scary change.

Comment (simonsmh) [inline run.sh]: See https://github.com/PokemonGoF/PokemonGo-Bot/commit/e03f8341bb536dea3bc9753d705ea21e20a31418
Someone need it. It's same as /n in echo.

Comment (simonsmh) [inline run.sh]: That's how read works, not a change but a fix of typo to make it show date in the next line.

**Comment by TheSavior** [comment]
:+1:

**Comment by devn0ll** [comment]
@simonsmh Thanks looks much better now

**Pull request merged**